source/lib.civet

```diff
@@ -876,52 +876,55 @@ function isExistence(exp) {
   }
 }
 
+// all relational operators could be chained in theory, including in and instanceof
+relationalOps := ["==", "===", "!=", "!==", "<", "<=", ">", ">=", "in"]
+
+function isRelationalOp(op)
+  relationalOps.includes(op.token) or op.relational
```

Comment:
I'd prefer to use a `Set` (and `.has`) here. (Maybe also for `lowerPrecedenceOps`)